README.md

```diff
@@ -4,14 +4,32 @@ Zenith substitutes PostgreSQL storage layer and redistributes data across a clus
 
 ## Running local installation
 
-1. Build zenith and patched postgres
+1. Install build dependencies and other useful packages
+
+On Ubuntu this set of packages should be sufficient to build the code:
+```text
+build-essential libreadline-dev zlib1g-dev flex bison libxml2-dev
+libcurl4-openssl-dev libssl-dev clang
+```
+
+[Rust] 1.48 or later is also required.
```

Comment:
May be add link to rustup here? Off-topic: I also checked what is reasonable way to install rust compiler in production (in the long term it is better not to install compiler on production servers at all, but it is handy to have it around for now) and it seems that people are just loading rustup-init binary for their architecture and run it.

Reply:
The link is there already, it's the `[Rust]` part.

I'm confused by the rest of this: why would we install the rust compiler in production? Wouldn't we just install binaries?

Reply:
Huh, didn't know about link references in github markdown.

> I'm confused by the rest of this: why would we install the rust compiler in production? Wouldn't we just install binaries?

Sorry for the confusion. I meant installing rust in places where you do care about compiler integrity. E.g. in a build job itself.

Reply:
I'm not sure if there are any CI "install rust" strategies that are better or worse than others. We're just grabbing a docker layer from CircleCI because it's convenient.